tests/src/performance/linkbench/linkbench.cs

```diff
@@ -155,75 +238,162 @@ public class LinkBench
         public static string AssetsDir;
         private static Benchmark CurrentBenchmark;
 
+        private static Benchmark[] Benchmarks =
+        {
+            new Benchmark("HelloWorld",
+                "LinkBench\\HelloWorld\\bin\\release\\netcoreapp2.0\\win10-x64\\unlinked",
+                "LinkBench\\HelloWorld\\bin\\release\\netcoreapp2.0\\win10-x64\\linked",
+                () => Benchmark.AddLinkerReference("LinkBench\\HelloWorld\\HelloWorld.csproj")),
+            new Benchmark("WebAPI",
+                "LinkBench\\WebAPI\\bin\\release\\netcoreapp2.0\\win10-x64\\unlinked",
+                "LinkBench\\WebAPI\\bin\\release\\netcoreapp2.0\\win10-x64\\linked",
+                () => { Benchmark.AddLinkerReference("LinkBench\\WebAPI\\WebAPI.csproj");
+                        Benchmark.PreventPublishFiltering("LinkBench\\WebAPI\\WebAPI.csproj"); }),
+            new Benchmark("MusicStore",
+                "LinkBench\\JitBench\\src\\MusicStore\\bin\\release\\netcoreapp2.0\\win10-x64\\unlinked",
+                "LinkBench\\JitBench\\src\\MusicStore\\bin\\release\\netcoreapp2.0\\win10-x64\\linked",
+                () => { Benchmark.AddLinkerReference("LinkBench\\JitBench\\src\\MusicStore\\MusicStore.csproj");
+                       Benchmark.SetRuntimeFrameworkVersion("LinkBench\\JitBench\\src\\MusicStore\\MusicStore.csproj"); }),
+            new Benchmark("MusicStore_R2R",
+                "LinkBench\\JitBench\\src\\MusicStore\\bin\\release\\netcoreapp2.0\\win10-x64\\R2R\\unlinked",
+                "LinkBench\\JitBench\\src\\MusicStore\\bin\\release\\netcoreapp2.0\\win10-x64\\R2R\\linked"),
+            new Benchmark("Corefx",
+                "LinkBench\\corefx\\bin\\ILLinkTrimAssembly\\netcoreapp-Windows_NT-Release-x64\\pretrimmed",
+                "LinkBench\\corefx\\bin\\ILLinkTrimAssembly\\netcoreapp-Windows_NT-Release-x64\\trimmed"),
+            new Benchmark("Roslyn",
+                "LinkBench\\roslyn\\Binaries\\Release\\Exes\\CscCore\\win7-x64\\publish",
+                "LinkBench\\roslyn\\Binaries\\Release\\Exes\\CscCore\\win7-x64\\Linked")
+        };
+
+        static int UsageError()
+        {
+            Console.WriteLine("Usage: LinkBench [--nosetup] [--nobuild] [--perf:runid <id>] [<benchmarks>]");
+            return -4;
```

Comment:
It would be nice to add descriptions of the flags, and a list of available benchmarks.